responses.py

```diff
@@ -553,6 +553,10 @@ def add_callback(
     def calls(self):
         return self._calls
 
+    @property
+    def matches(self):
```

Comment:
Why not a standard function? That would be more consistent with the other parts of responses and fit with the module auto-exporter.

Reply:
I was matching the implementation of `calls` above, as they seemed to be similar grouping of functionality. On review I worry that `matches` is too confusing a name because ``BaseResonse`` already has a `matches()` function. Do you have a preferred name or method signature?

Reply:
Do we need a public method? Are you frequently dumping the mocks you created?

Reply:
I found it useful for debugging. Especially when initially getting used to the library I found I was getting issues with trying to call a response that I thought I had added but had no way to be sure. Being able to check the `_matches` greatly sped up the search for the issue (which was with our codebase, not the library)

Due to the auto-exporter not exporting private members (`_matches`) it's the difference between inspecting `responses.matches` and `responses.mock._matches` during debugging. Not a huge issue once you know about it but not very accessible and not easy to find for people just starting out with the lib that might run into issues. If you'd prefer not to have the public method and just update the docs I can do that also.

Reply:
For now I think updating the docs is ok. If this situation comes up more we can expand the public interface.